src/main/java/com/leaf/billing/utils/SparkCommands.java

```diff
@@ -31,4 +36,43 @@ public static LocalDate locateOldestTimestampGroup(String path) throws IOExcepti
         return currentOldest;
     }
 
+    public static Dataset<Row> loadParquetFilesBasedOnDate(SparkSession sparkSession, String folderPath,
```

Comment:
```suggestion
    public static Dataset<Row> loadParquetFilesForSpecifiedDate(SparkSession sparkSession, String folderPath,
```
I took me a little bit to understand that this is for the specific date, not for a range or before/after the timestamp.